### ISSUE TITLE
Cannot read properties of undefined (reading 'player')

### ISSUE THREAD

**Issue by plopilpy** [esmBot/esmBot#313]
### Command that caused the error

/music queue OR /music nowplaying OR /music skip

### Steps to reproduce

1. Use any music command EXCEPT /music play

### Self-hosted instance?

No

### Error file

```
Message: TypeError: Cannot read properties of undefined (reading 'player')
    at NowPlayingCommand.run (file:///home/esmBot/esmBot/commands/music/nowplaying.js?v=101:10:36)
    at MusicAIOCommand.run (file:///home/esmBot/esmBot/commands/music/music.js?v=100:18:34)
    at default (file:///home/esmBot/esmBot/events/interactionCreate.js:27:39)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Stack Trace: TypeError: Cannot read properties of undefined (reading 'player')
    at NowPlayingCommand.run (file:///home/esmBot/esmBot/commands/music/nowplaying.js?v=101:10:36)
    at MusicAIOCommand.run (file:///home/esmBot/esmBot/commands/music/music.js?v=100:18:34)
    at default (file:///home/esmBot/esmBot/events/interactionCreate.js:27:39)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

### Additional context

None!

**Comment by TheEssem** [comment]
I've seen this come up a bit recently, might I ask what the state of the player was when you ran the command? Was the bot already trying to play something in voice chat?

**Comment by TheEssem** [comment]
Also seemingly related to #309

**Comment by TheEssem** [comment]
Should be fixed by 0c08422
